Dockerfile

```diff
@@ -7,5 +7,5 @@ RUN apt-get update && apt-get install -y gcc python-dev libgmp3-dev
 COPY requirements.txt /requirements.txt
 RUN pip install -r requirements.txt --upgrade --upgrade-strategy eager
 
-COPY pontis-package /pontis-package
-RUN pip install -e pontis-package
+ARG PONTIS_PACKAGE_VERSION
```

Comment:
My question is, if ```ARG PONTIS_PACKAGE_VERSION``` is coming from pontis-package/setup.cfg is that not still semantically similar to pulling the local download of ```COPY pontis-package /pontis-package``` since we are just using the version pragma?

I get how dropping the -e flag would remove editable mode (does it? I'm inferring; reading the pip install man file there is no -e flag and google autocomplete shows I searched it before to no avail :/), but since you're grepping on local machine wouldn't this code just build to whatever the latest local pontis package version is? I think that's a fine assumption to make, just making sure that I'm not missing something about a global npm-like latest install of pontis-package that is actually happening?

I realize I probably sound really stupid here, because I don't see the diff besides being cleaner, but here's to not being afraid to sound stupid  🍻

Reply:
Ok had a final idea here, which made me comfortable posting above. Since pypi is analogous to npm, is this change ```RUN pip install pontis-package==$PONTIS_PACKAGE_VERSION``` vs doing just pontis-package:latest  or pip install pontis-package necessary for us (and repo users) to identify which entry to pull from the pypi index base? That would make more sense.

Follow up clarifying question, where in the code are we publishing the pontis-package to pypi or updating that index. Is that a GHA? Does PyPi just watch these repo for stable releases. Thanks for helping me through these questions, have no experience publishing node or py packages so apologies in advance

Reply:
^^Answer to this is in the ```Pontis Package``` section in the README, specifically using twine as a separate process to upload package to pypi